### PR TITLE
add params to import xpriv

### PR DESCRIPTION
**Pull request by matiu**

### CODE REVIEW

File: lib/credentials.js [hunk 1]
```diff
@@ -88,9 +88,20 @@ Credentials.createWithMnemonic = function(network, passphrase, language, account
   return x;
 };
 
-Credentials.fromExtendedPrivateKey = function(xPrivKey) {
+Credentials.fromExtendedPrivateKey = function(xPrivKey, derivationStrategy, addressType) {
```
Comment (isocolsky) [inline lib/credentials.js]: Shouldn't we also include `account`?

Comment (matiu) [inline lib/credentials.js]: Yes!

**Comment by matiu** [comment]
Updated. Adds `account`, adds params also to API. endpoint and fix tests and documentation

**Comment by matiu** [comment]
updated.

**Pull request merged**